modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="installing-with-usb-media_{context}"]
+
+= Installing with USB media
+
+Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node to install singe node OpenShift (SNO) via the Assisted Installer (AI).
+
+.Procedure
+
+. On the administration node, insert a USB drive into a USB port.
+
+. Create a bootable USB drive.
```

Comment:
When the instructional text introduces a command, end the instructional text with a colon.

That will match the convention that is applied to [code blocks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) in the guidelines.